source/test/Config.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 Klemens D. Morgenstern
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
It seems I have to repeat myself. We are not using the Boost license, Please follow the example set in the other files.

Reply:
Sorry, IDO template